v1.0.0.yaml

```diff
@@ -2952,203 +2823,913 @@ paths:
-                - 30
-                - 40
-                - true
-            defaultValue:
-              description: |
-                The default value for this custom metadata field. Date type of default value depends on the field type.
-            isValueRequired:
+            add_shadow:
               type: boolean
-              description: |
-                Specifies if the this custom metadata field is required or not.
-            minValue:
-              description: |
-                Minimum value of the field. It is only set if field type is `Date` or `Number`. For `Date` type field, the value will be in ISO8601 string format. For `Number` type field, it will be a numeric value.
-            maxValue:
-              description: |
-                Maximum value of the field. Only set this property if `type` is set to `Date` or `Number`. For `Date` type field, the value will be in ISO8601 string format. For `Number` type field, it will be a numeric value.
-            minLength:
-              description: |
-                Minimum length of string. Only set this property if `type` is set to `Text` or `Textarea`.
-              type: number
-            maxLength:
-              description: |
-                Maximum length of string. Only set this property if `type` is set to `Text` or `Textarea`.
-              type: number
-    fileDetails:
+              description: >
+                Whether to add an artificial shadow to the result. Default is false.
+                Note: Adding shadows is currently only supported for car photos.
+              default: false
+            semitransparency:
+              type: boolean
+              description: >
+                Allows semi-transparent regions in the result. Default is true.
+                Note: Semitransparency is currently only supported for car windows.
+              default: true
+            bg_color:
+              type: string
+              description: >
+                Specifies a solid color background using hex code (e.g., "81d4fa", "fff") or color name (e.g., "green").
+                If this parameter is set, `bg_image_url` must be empty.
+            bg_image_url:
+              type: string
+              description: >
+                Sets a background image from a URL. If this parameter is set, `bg_color` must be empty.
+    AutoTaggingExtension:
+      title: Auto tagging
+      type: object
+      required:
+        - name
+        - minConfidence
+        - maxTags
+      properties:
+        name:
+          type: string
+          enum: ['google-auto-tagging', 'aws-auto-tagging']
+          description: Specifies the auto-tagging extension used.
+        minConfidence:
+          type: integer
+          description: Minimum confidence level for tags to be considered valid.
+        maxTags:
+          type: integer
+          description: Maximum number of tags to attach to the asset.
+  responses:  
+    FileDetails:
```

Comment:
- Missing `embeddedMetadata` property in response
- isPrivateFile, customMetadata should also be marked as nullable or optional since these fields are only present in the response when specifically request by the `responseFields` parameter.